apps/site/src/lib/common/components/Nav/Nav.tsx

```diff
@@ -32,7 +33,15 @@ function AppNavbar() {
   return (
     <header className={styles.header}>
       {/* eslint-disable-next-line @next/next/no-img-element */}
-      <img className={styles.logo} src={Logo.src} alt="hack logo" />
+      <Link href="/">
+        <Image
```

Comment:
Image component here probably isn't necessary since the source is already an svg. Either revert back to img tag or remove the comment above.

Reply:
What is the use case of the Next Image component? I just used it cause when I was learning thats what they told me to do cause of SSR efficiency or something like. Like when should I use it and not? I'll probably just remove the comment as well.

Reply:
The Next Image component optimizes raster images (png, jpeg, etc.). It basically resizes them so that you aren't loading unnecessarily large images. However, in this case, it's not necessary since the file is a vector file (.svg), meaning the size stays constant regardless the size.

Reply:
oh i see i see. Thanks for the explanation!